test/modules/packages/Crypto/saru.skipif

```diff
@@ -0,0 +1,2 @@
+# skip non-linux64 platforms for now
+CHPL_TARGET_PLATFORM != linux64
```

Comment:
Executable skipif that checks if openssl is installed?

Reply:
On OSX there is an OpenSSL installed, but not compatible with the homebrew version.  So it needs to be installed and be the homebrew one on Macs.  So I just said I'll skip non-Linux for now.

Reply:
Ah ok.

In addition to skipping on non-linux then should we also skip if openssl inn't installed, or do we expect every linux system to have it?

Reply:
For now I feel fairly safe saying that Linux systems that we test on have a reasonably standard (though sometimes very old) OpenSSL installed.